File: php/sample.php
<?php

$curl = curl_init();

curl_setopt_array($curl, [
	CURLOPT_URL => "https://taobao-api2.p.rapidapi.com/taobao/v1/item/603395007187/detail",
	CURLOPT_RETURNTRANSFER => true,
	CURLOPT_FOLLOWLOCATION => true,
	CURLOPT_ENCODING => "",
	CURLOPT_MAXREDIRS => 10,
	CURLOPT_TIMEOUT => 30,
	CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
	CURLOPT_CUSTOMREQUEST => "GET",
	CURLOPT_HTTPHEADER => [
		"x-rapidapi-host: taobao-api2.p.rapidapi.com",
		"x-rapidapi-key: <YOUR API KEY>"
	],
]);

$response = curl_exec($curl);
$err = curl_error($curl);

curl_close($curl);

if ($err) {
	echo "cURL Error #:" . $err;
} else {
	echo $response;
}

File: c/sample.c
CURL *hnd = curl_easy_init();

curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "GET");
curl_easy_setopt(hnd, CURLOPT_URL, "https://taobao-api2.p.rapidapi.com/taobao/v1/item/603395007187/detail");

struct curl_slist *headers = NULL;
headers = curl_slist_append(headers, "x-rapidapi-key: <YOUR RAPIDAPI KEY>");
headers = curl_slist_append(headers, "x-rapidapi-host: taobao-api2.p.rapidapi.com");
curl_easy_setopt(hnd, CURLOPT_HTTPHEADER, headers);

CURLcode ret = curl_easy_perform(hnd);
